src/OrbitGl/CaptureViewElementTest.cpp

```diff
@@ -171,15 +171,20 @@ TEST(CaptureViewElement, MouseWheelEventRecursesToCorrectChildren) {
       .WillRepeatedly(Return(CaptureViewElement::EventResult::kHandled));
 
   EXPECT_EQ(CaptureViewElement::EventResult::kIgnored,
```

Comment:
For now there are two methods that we might be interested to test this functionality: OnMouseWheel and OnMouseMove. I wouldn't Mock OnMouseMove because I want to test functionality of OnMouseEnter and OnMouseLeave. I can create a new Mocked class but I don't find a lot of value so I will leave as it is. As soon as we start using other methods for picking I will extend this test.